### ISSUE TITLE
Master is broken

### ISSUE THREAD

**Issue by schneems** [heroku/heroku-buildpack-activestorage-preview#15]
```
$ cd /tmp
$ git clone https://github.com/sharpstone/active_storage_local.git
$ cd active_storage_local/
$ heroku create
$ heroku buildpacks:set https://github.com/heroku/heroku-buildpack-activestorage-preview
$ heroku buildpacks:add heroku/ruby
$ git push heroku master
Enumerating objects: 1, done.
Counting objects: 100% (1/1), done.
Writing objects: 100% (1/1), 185 bytes | 185.00 KiB/s, done.
Total 1 (delta 0), reused 0 (delta 0)
remote: Compressing source files... done.
remote: Building source:
remote:
remote: -----> ActiveStorage Preview app detected
remote: -----> Installing binary dependencies for ActiveStorage Preview
remote:        Detected ffmpeg version change, flushing cache
remote:        Downloading packagesE: List directory /app/tmp/cache/.heroku/activestorage-preview/state/lists/partial is missing. - Acquire (2: No such file or directory)
remote:  !     Push rejected, failed to compile ActiveStorage Preview app.
remote:
remote:  !     Push failed
remote: Verifying deploy...
remote:
remote: ! Push rejected to immense-headland-78006.
remote:
To https://git.heroku.com/immense-headland-78006.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://git.heroku.com/immense-headland-78006.git'
```

**Comment by kylejtay** [comment]
I'm seeing the same

**Comment by cji** [comment]
Sorry about that - fix is in #16
